src/components/Dspaces/DspacePosts.jsx

```diff
@@ -0,0 +1,191 @@
+import React, { Component } from "react";
+import { Typography, Divider, CircularProgress } from "@material-ui/core";
+import { Container, Row } from "react-bootstrap";
+import Post from "../Post";
+import {database} from '../../firebase/firebase.utils';
+
+
+import Box from '@material-ui/core/Box';
+import SortIcon from '@material-ui/icons/Sort';
+import Button from '@material-ui/core/Button';
+import Menu from '@material-ui/core/Menu';
+import MenuItem from '@material-ui/core/MenuItem';
+
+let posts = [];
+
+
+export class DspacePosts extends Component {
+
```

Comment:
I believe we have used similar functions for displaying the posts on the home screen, and are we repeating those here? because if we are then we aren't following the DRY principle. 
I might be wrong, can you please clarify? @akylus